src/main/java/appeng/client/gui/widgets/GuiScrollbar.java

```diff
@@ -25,6 +25,10 @@
 
 public class GuiScrollbar implements IScrollSource
 {
+	private String txtBase = "minecraft";
+	private String txtFile = "gui/container/creative_inventory/tabs.png";
```

Comment:
static final maybe

Reply:
in [GuiPatternTermEx.java](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/143/files#diff-432838dd0beada7a49e638133370b2c943c12de23f9cb63402ec3aa7a6db3a32) line 44 i change it for small scrollbar